front/lib/api/llm/clients/google/types.ts

```diff
@@ -1,4 +1,15 @@
-export const GOOGLE_AI_STUDIO_WHITELISTED_MODEL_IDS = ["gemini-2.5-pro"];
+import type { ModelIdType } from "@app/types";
+import { GEMINI_2_5_PRO_MODEL_ID } from "@app/types";
+
+export const GOOGLE_AI_STUDIO_WHITELISTED_MODEL_IDS = [
+  GEMINI_2_5_PRO_MODEL_ID,
+] as const;
 
 export type GoogleAIStudioWhitelistedModelId =
   (typeof GOOGLE_AI_STUDIO_WHITELISTED_MODEL_IDS)[number];
+
+export function isGoogleAIStudioWhitelistedModelId(
```

Comment:
that's a good idea, I would just have put that in another PR to ease the review of this one